### ISSUE TITLE
[BUG] Accessibility Issues with Pagination

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#504]
**Describe the bug**

**To Reproduce**
1. Go to https://modus-web-components.trimble.com/?path=/docs/components-pagination--default
2. Inspect Element, the paginations are using `<ol>` ordered lists but the page numbers don't match the list order due to there being a `<` button. The current selected page doesn't have the aria-current="page" attribute.

**Expected behavior**
- We should use `<ul>` instead like Modus Bootstrap does.
- The current page should have the `aria-current="page"` attribute (see https://modus-bootstrap.trimble.com/components/pagination/ for reference)

**Comment by msankaran0712** [comment]
@ryan-henness-trimble any update?
